frontend/src/lib/components/SortableDragIcon.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+export function SortableDragIcon(): JSX.Element {
+    return (
+        <span role="img" aria-label="drag" className="anticon">
+            <svg width="1em" height="1em" viewBox="0 0 16 16" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Longer term I think we should think about standardizing our icons to a consistent `viewbox` size. This will help us have a more predictable layout across instances, and it'll help give us a foundation for larger touch targets on actionable icons. 24px square is usually a solid starting point, but I don't have enough context to say we should change this now.

Reply:
The size here is assigned by width and height; because it's an `em` measurement it will scale with the font-size of its parent element. It is not 16px square. The svg viewbox property is like a set of "anchor points" by which sizes/positions of paths inside are scaled.